ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -344,7 +344,10 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
                                 namespace: application.metadata.namespace
                             });
                             return (
-                                <div className={`application-details ${this.props.match.params.name}`}>
+                                <div
+                                    className={`application-details ${
+                                        typeof application.metadata.labels !== 'undefined' ? application.metadata.labels[appModels.LabelCustomCSSKey] : this.props.match.params.name
```

Comment:
I think we should hard-code a prefix on the class name so people can't hijack existing class names and bork the rendering.

Reply:
![Screenshot 2023-06-16 at 12 16 34 PM](https://github.com/argoproj/argo-cd/assets/11219262/01fca3de-9398-46d5-9b40-535dbe878c86)